src/hearings/containers/hearing-actuals/hearing-actuals-final-confirmation/hearing-actuals-final-confirmation.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { HearingConfirmationSource } from 'src/hearings/models/hearings.enum';
+import * as fromHearingStore from '../../../store';
+
+@Component({
+  selector: 'exui-hearing-actuals-final-confirmation',
+  templateUrl: './hearing-actuals-final-confirmation.component.html'
+})
+export class HearingActualsFinalConfirmationComponent implements OnInit {
+
+  public heading: string;
+  public subheading: string;
+  public caseId: string;
+
+  constructor(protected readonly hearingStore: Store<fromHearingStore.State>) {
+  }
+
+  public get hearingConfirmationSourceEnum() {
+    return HearingConfirmationSource;
+  }
```

Comment:
this probably can be removed

Reply:
The text with link differs between the hearing request confirmation and hearing actuals confirmation. I had to use this to display the correct link text.

Reply:
I have removed this code as it is no more required.